src/Tasks/Microsoft.NET.Build.Tasks/NugetContentAssetPreprocessor.cs

```diff
@@ -26,7 +27,8 @@ public bool Process(string originalAssetPath, string relativeOutputPath, out str
         {
             if (_preprocessedOutputDirectory == null)
             {
-                throw new Exception($"{nameof(NugetContentAssetPreprocessor)} should be configured before any assets are processed");
+                Debug.Fail($"{nameof(NugetContentAssetPreprocessor)} should be configured before any assets are processed");
+                throw new InvalidOperationException();
```

Comment:
I was trying to make it obvious that we don't need localization, but now I think it's easier if we just go ahead and put all exception messages in a resource so that we can continue to audit all throw and log calls without exceptions.